app/build.gradle

```diff
@@ -12,12 +12,12 @@ android {
 
     defaultConfig {
         applicationId "ch.epfl.sdp.appart"
-        minSdkVersion 23
+        minSdkVersion 24
```

Comment:
Are we allowed to bump the minSdkVersion ?

Reply:
I Hope because some interesting feature of Java 8 were allowed only in sdk 24...